Packs/FeedCyrenThreatInDepth/Scripts/CyrenThreatInDepthRelatedWidget/CyrenThreatInDepthRelatedWidget.py

```diff
@@ -0,0 +1,84 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+from datetime import datetime
+
+
+SCORE_TO_REPUTATION_TEXT = {
+    Common.DBotScore.NONE: f"None ({Common.DBotScore.NONE})",
+    Common.DBotScore.GOOD: f"Good ({Common.DBotScore.GOOD})",
+    Common.DBotScore.SUSPICIOUS: f"Suspicious ({Common.DBotScore.SUSPICIOUS})",
+    Common.DBotScore.BAD: f"Bad ({Common.DBotScore.BAD})",
+}
+
+
+def create_relationship_object(value: str, relationship_type: str, indicator_type: str,
+                               timestamp: str, entity_category: str, reputation: int = Common.DBotScore.NONE):
+    try:
+        timestamp_parsed = datetime.fromisoformat(timestamp.replace("Z", "+00:00"))
+        timestamp_human_readable = timestamp_parsed.strftime("%Y-%m-%d, %H:%M:%S")
+    except ValueError:
+        demisto.info("could not parse timestamp %s, keeping original string", timestamp)
+        timestamp_human_readable = timestamp
+
+    indicator_object = {
+        "Value": f"{value}\n",
+        "Reputation": SCORE_TO_REPUTATION_TEXT[reputation],
+        "Relationship Type": relationship_type,
+        "Indicator Type": indicator_type,
+        "Timestamp UTC": timestamp_human_readable,
+        "Entity Category": entity_category,
+    }
+
+    return indicator_object
+
+
+def cyren_feed_relationship(args) -> CommandResults:
+    indicator = args["indicator"]
+    relationships = indicator.get("CustomFields", {}).get("cyrenfeedrelationships", []) or []
```

Comment:
Is there a chance that `args` will not have `indicator` key? If there is such a chance, let's check it before calling `indicator.get` .
```suggestion
    indicator = args.get("indicator")
    if indicator:
        relationships = indicator.get("CustomFields", {}).get("cyrenfeedrelationships", []) or []
   else:
        `choose what you would like to do in this case`
```

Reply:
It is probably a good idea. Although this is a script with a tag of `dynamic-indicator-widget` and I suspect the software to guarantee the property.

Reply:
done